andela/travel_tool_back
# #160820759 Send Email Notification for Posted Comment

#### What does this PR do?
send email notification for a posted comment

#### Description of Task to be completed?
- create an email notification engine
- send email notification for posted comment by the requester.
- send email notification for posted comment by the manager

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_back.git
- Enter into the project directory - cd travel_tool_back
- Install dependencies and start the server - make start
- Start the frontend.
- Make a request and choose yourself as the manager
- Post a comment on the request details
- Check your gmail account for a new message from travela regarding the posted comment
- Click on the view request button to redirect to the request

#### Any background context you want to provide?
- In other to test this feature, you need to update your role on the Users table to that of a manager.
Manager roleId: 50319
- Check the `env.example` to check the variable that is required to implement this task which is `SENDGRID_API_KEY`, `REACT_APP_AUTH_REDIRECT_URL` and `APP_EMAIL`

#### What are the relevant pivotal tracker stories?
[#160820759](https://www.pivotaltracker.com/story/show/160820759)

#### Screenshots (if appropriate)
<img width="688" alt="screen shot 2018-10-04 at 12 00 04 pm" src="https://user-images.githubusercontent.com/28366272/46470034-218e5800-c7cd-11e8-83a0-24909c201548.png">


#### Questions:
N/A
